Android/app/src/main/java/com/mehequanna/mmiw/MmiwActivity.kt

```diff
@@ -78,9 +80,15 @@ class MmiwActivity : AppCompatActivity() {
         }
 
         capture_button.setOnClickListener {
-            capture_button.isEnabled = false
-            sceneView.pause()
-            takePhoto() // TODO Move this until after the image is ready
+            changeButtonVisibility(false)
+        }
+
+        send_button.setOnClickListener {
```

Comment:
This is confusing to me... why are we waiting to take the picture until send is clicked... don't we need to take the photo when the capture button is clicked?

Reply:
I got this idea from @callum-code . Callum suggested just pausing ARCore and then working with the scene from there. Then copy when the user was ready to send. I thought this was brilliant since creating a bitmap and file are very expensive operations, also it negates the need to create a whole view to show the bitmap after it's created. From my testing it's seems to make a pretty fluid experience.

I could rename the function to `createPhoto()` or `sharePhoto()`?

Reply:
I gues that makes sense but it seems like the only action that happens right now when you hit the capture_button is to changeButtonVisibility

Reply:
I could call it prepareImage(). It pauses ArSceneview. I will also have to add the header and carousel view. But yeah capture is less about capturing at this point.

Reply:
It's a little odd to me to hide the sceneView.pause inside of the changeButtonVisibility because it's not really related to the button visibility... I get that they are closely tied but maybe change the name of the method or else have a separate pauseSceneView(paused: Boolean) method ?

Reply:
Ah, yeah. I can totally do that. Sorry I think I misunderstood the problem.